pages/docs/basic-features/layouts.mdx

```diff
@@ -28,7 +30,7 @@ export default function Layout({ children }) {
       <main>{children}</main>
       <Footer />
     </>
-  )
+  );
```

Comment:
We should keep the coding style from the origin docs, please consider disable the auto formatter during translation!

Reply:
Fixed